horizons/__init__.py

```diff
@@ -0,0 +1,281 @@
+from dataclasses import dataclass
+from datetime import datetime, timedelta
+import dateutil.parser
+from common.minimodel import Target, TargetTag, Site
+import numpy as np
+import numpy.typing as npt
+import requests
+from typing import Tuple
+
+import os
+
+@dataclass
+class EphemerisCoordinates:
+    """
+    Both ra and dec are in radians.
+
+    """
+    ra: npt.NDArray[float]
+    dec: npt.NDArray[float]
+    time: npt.NDArray[float]
+
+    @staticmethod
+    def dms2deg(dms):
+
+        if dms is None:
+            return None
+
+        d = float(dms[0])
+        m = float(dms[1])
+        s = float(dms[2])
+        sign = dms[3]
+        dd = d + m / 60. + s / 3600.
+
+        if sign == '-':
+            dd *= -1.
+
+        return dd
+
+    @staticmethod
+    def dms2rad(dms):
+        if dms is None:
+            return None
+        
+        dd = EphemerisCoordinates.dms2deg(dms)
+        rad = dd * np.pi / 180.
+        return rad
+
+    @staticmethod
+    def hms2rad(hms):
+        if hms is None:
+            return None
+
+        h = float(hms[0])
+        m = float(hms[1])
+        s = float(hms[2])
+        hours = h + m / 60. + s / 3600.
+        rad = hours * np.pi / 12.
+
+        return rad
```

Comment:
This is all good stuff that we should probably move out of here and put in a package in `common.helpers` or something similar.